server/src/main/proto/spine/system/server/c/command_lifecycle_commands.proto

```diff
@@ -55,15 +56,36 @@ message AcknowledgeCommand {
     spine.core.CommandId id = 1;
 }
 
+// A command to schedule a command.
```

Comment:
`A system command to schedule a domain command.` And this the very least. I'd even say more.